public/index.html

```diff
@@ -15,6 +15,35 @@
     <title>Survey Taker Application</title>
   </head>
   <body>
+    <!-- Navigation Bar -->
+        <div class="nav-container">
+          <nav class="navbar navigation-bar" role="navigation" aria-label="main navigation">
+              <div class="navbar-brand">
+                <a class="navbar-item" href="#">
+                  <img src="../images/logo.png"  alt="GirlScript Banglore Logo">
+                  <h1 class="nav-title navbar-item"></h1>
+                </a>
+                <div class="navbar-burger burger" data-target="navbarExampleTransparentExample">
```

Comment:
Girlscript logo is not coming correctly. Please have a look into it

Reply:
Maam I have tried correcting it but I am unable to debug it. I don't know whats wrong
```suggestion
                <a class="navbar-item" href="#">
                  <img src="assets/images/logo.png"  alt="GirlScript Banglore Logo">
                  <h1 class="nav-title navbar-item"></h1>
                </a>
                <div class="navbar-burger burger" data-target="navbarExampleTransparentExample">
```
I have used this code as well but it doesn't work.

Reply:
I will give u a quick fix here. Do one thing  upload the logo of GS Blr on this website https://imgbb.com/
It will give u a link for the logo. Use that logo directly here, instead of adding a logo in assets file

Reply:
why its not working i will tell u the reason, this assets folder is not the part of the react app, its the part of github repo. The assets folder for the react-app should be either inside src folder or this public folder.

Reply:
The other solution can be making an assets folder inside the src folder and then adding the logo.

Reply:
Do one thing upload the logo of GS Blr on this website https://imgbb.com/
It will give u a link for the logo. I have used this. Thank you :)